content/SUMMARY.md

```diff
@@ -50,27 +51,3 @@
     * [Codeception](/testing/automation-using-codeception.md)
   * [Git integration](testing/git-integration.md)
   * [Manual Testing Guide](testing/testing-data.md)
-* [Build](build/index.md)
```

Comment:
@chandrapatel are we removing build, deployment, CLI and other sections?

Reply:
@rahul286 Nope. I've removed it from `Table of Contents` because it's incomplete and outdated. We will add it back once those pages updated.

Reply:
Fine. Actually, Gitbook uses SUMMARY.md file to generate pages. So these sections won't be generated after this change get merged. 

Entire sections like this https://wpveda.com/build/ may get removed. I will suggest not to remove sections from SUMMARY.md

Reply:
No issues. 🙂